app/Http/Controllers/HomeController.php

```diff
@@ -34,22 +34,23 @@ public function create()
     {
         return view('event_create', compact('user'));
     }
-    
-    // public function post(Request $request)
-    // {
-    //     // Auth::user()->events()->create()
-    //     $event = new Event;
-    //     // dd($event);
-    //     $event->user_id = Auth::user()->id;
-    //     $event->event_title = $request->input('event_title');
-    //     $event->representative_name = $request->input('representative_name');
-    //     $event->date = $request->input('event_date');
-    //     $event->area = $request->input('area');
-    //     $event->category = $request->input('Category');
-    //     $event->event_info = $request->input('event_info');
-    //     $event->image = 'image';
-    //     $event->save();
-    //     //Auth::user()->events()->save($event);
-    //     return redirect('/my_profile');
-    // }
+
+    public function post(Request $request)
+    {
+        // Auth::user()->events()->create()
+        $event = new Event;
+        // dd($event);
+        $event->user_id = Auth::user()->id;
+        $event->event_title = $request->input('event_title');
+        $event->representative_name = $request->input('representative_name');
+        $event->date = $request->input('event_date');
+        $event->area = $request->input('area');
+        $event->category = $request->input('Category');
+        $event->event_info = $request->input('event_info');
+        $event->image = 'image';
+        $event->save();
+        //Auth::user()->events()->save($event);
```

Comment:
Again, remove if you do not need anymore.

Reply:
its Delete.